redis/src/cluster_async/mod.rs

```diff
@@ -625,6 +625,23 @@ impl From<ArcStr> for OperationTarget {
     }
 }
 
+/// Represents a node to which a `MOVED` or `ASK` error redirects.
+pub(crate) struct RedirectNode {
+    /// The address of the redirect node.
+    pub _address: String,
+    /// The slot of the redirect node.
+    pub _slot: u16,
+}
+
+impl RedirectNode {
+    pub(crate) fn from_option_tuple(option: Option<(&str, u16)>) -> Option<Self> {
```

Comment:
Add a comment (mainly: what does the `option` contain)

Reply:
added